app/views/catalog/_osd_or_embed_default.html.erb

```diff
@@ -1,5 +1,8 @@
+<% @viewer = Viewer.find_by(exhibit: @exhibit) %>
```

Comment:
@jkeck Since the partial seems to be set through config in the catalog_controller (https://github.com/sul-dlss/exhibits/blob/add_mirador_as_viewer/app/controllers/catalog_controller.rb#L61-L73), I'm not sure how one would pass a local to this partial. Any advice?